apis/DeviceInfo/IDeviceInfo.h

```diff
@@ -38,6 +38,7 @@ namespace Exchange {
         virtual uint32_t DeviceType(string& deviceType /* @out */) const = 0;
         virtual uint32_t SocName(string& socName /* @out */) const = 0;
         virtual uint32_t DistributorId(string& distributorId /* @out */) const = 0;
+        virtual uint32_t Brand(string& brand /* @out */) const = 0;
```

Comment:
`BrandName` or `Brand` ?

Reply:
function name in Interface and implementation is Brand(), API in curl is brandname. followed the modelname implementation here.